src/NuGet.Core/NuGet.Protocol/PublicAPI/net472/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,2 @@
-#nullable enable
```

Comment:
The current dev branch has `#nullable enable`. I think we should add it back for all the 3 public api files modified in this PR.
https://github.com/NuGet/NuGet.Client/blob/80c3c74ca5c6d67912a19833e8cc2b10f875dc95/src/NuGet.Core/NuGet.Protocol/PublicAPI/netstandard2.0/PublicAPI.Unshipped.txt#L1